fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -462,7 +464,7 @@ Future<Optional<std::string>> createMetacluster(Reference<DB> db, ClusterName na
 			state Future<Optional<MetaclusterRegistrationEntry>> metaclusterRegistrationFuture =
 			    MetaclusterMetadata::metaclusterRegistration().get(tr);
 
-			wait(managementClusterCheckEmpty(tr));
+			state Future<Void> metaClusterEmptinessCheck = managementClusterCheckEmpty(tr);
```

Comment:
nit: metacluster is one word so the c can be uncapitalized:

```suggestion
			state Future<Void> metaclusterEmptinessCheck = managementClusterCheckEmpty(tr);
```